drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -684,6 +685,17 @@ static void set_up_fixed_clock_sources(void)
 		LL_RCC_HSI48_Enable();
 		while (LL_RCC_HSI48_IsReady() != 1) {
 		}
+
+		if (IS_ENABLED(STM32_HSI48_CRS_USB_SOF)) {
+			LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_CRS);
```

Comment:
Not an ideal solution, but please move this to `config_enable_default_clocks` in `drivers/clock_control/clock_stm32g0.c` (under whatever #ifdef's are required).

Reply:
Thanks for the review. I added it in the `drivers/clock_control/clock_stm32_ll_common.c` because I believe it applies to more STM32 families. I have personally used CRS with HSI48 on G0, L4 and H7 series, using that 3 lines in the board file (and one more when syncing on LSE instead).

I have the impressions that all families with HSI48 have a CRS, but I don't have a full view of all the families.

Reply:
I was refering to `LL_APB1_GRP1_EnableClock(LL_APB1_GRP1_PERIPH_CRS);`
Sorry if it wasn't clear enough